ops/docker-compose.yml

```diff
@@ -105,6 +105,12 @@ services:
       ROLLUP_ENFORCE_FEES: ${ROLLUP_ENFORCE_FEES:-true}
       ROLLUP_FEE_THRESHOLD_DOWN: 0.9
       ROLLUP_FEE_THRESHOLD_UP: 1.1
+
+      # private key: a6aecc98b63bafb0de3b29ae9964b14acb4086057808be29f90150214ebd4a0f
+      SYSTEM_ADDRESS_0_DEPLOYER: "0xa961b0d6dce82db098cf70a42a14add3ee3db2d5"
+
+      # private key: 3b8d2345102cce2443acb240db6e87c8edd4bb3f821b17fab8ea2c9da08ea132
+      SYSTEM_ADDRESS_1_DEPLOYER: "0xdfc82d475833a50de90c642770f34a9db7deb725"
```

Comment:
Consider indicating why it's safe to publish these keys. Like [this](https://github.com/ethereum-optimism/optimism/blob/fb70e4cd2512b3259224bf5ba534e29874bcbb70/ops/docker-compose-nobuild.yml#L97-L98) would be sufficient.